### PR TITLE
`estuary setup`

### LINKED ISSUE #11: Estuary should help the host with creating an initial admin account

### PR DESCRIPTION
**Pull request by elijaharita**
Adds `estuary setup` which sets up an initial auth token. 

Closes #11

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -204,6 +205,48 @@ func main() {
...
+					PassHash: passHash,
+					Perm:     100,
+				}
+				if err := db.Create(newUser).Error; err != nil {
+					return fmt.Errorf("admin user creation failed: %w", err)
+				}
+
+				authToken := &AuthToken{
+					Token:  "EST" + uuid.New().String() + "ARY",
+					User:   0,
```
Comment (whyrusleeping) [inline main.go:236]: 0 should be `newUser.ID` instead.

File: main.go [hunk 2]
```diff
@@ -204,6 +205,48 @@ func main() {
...
+				}
+				if err := db.Create(authToken).Error; err != nil {
+					return fmt.Errorf("admin token creation failed: %w", err)
+				}
+
+				fmt.Printf("Auth Token: %v", authToken.Token)
+
+				return nil
+			},
+			Usage: "Creates an initial auth token under new user \"admin\"",
```
Comment (whyrusleeping) [inline main.go:247]: nit: put the usage above the action function definition

Comment (elijaharita) [reply main.go:247]: LOL whoops, that's not a nit at all, i committed a crime

File: main.go (hunk 1 above)
Comment (elijaharita) [reply main.go]: ok, fixes applied

**Review by whyrusleeping — APPROVED** [review]

**Comment by whyrusleeping** [comment]
Great, this makes setting up your own estuary deployment a good deal easier!

**Pull request merged**